web/app.fluidity.money/app/components/MobileModal.tsx

```diff
@@ -149,6 +150,13 @@ export default function MobileModal({
 
             {/* Navigation Buttons */}
             <div className="mobile-navbar-right">
+              {/* Chain Switcher */}
+              {showMobileNetworkButton ? (
+                  <ChainSelectorButton
+                  chain={chains[network as "ethereum" | "solana"]}
```

Comment:
Hey okay this makes sense. I've implemented this change for `MobileModal.tsx` and `dashboard.tsx` since I'm familiar with how the files works. Are you saying you want me to go through the rest of the codebase though and change the network type to ChainName across all the app, or just all the references in `MobileModal.tsx` and `dashboard.tsx`?

Reply:
just updating the references here is fine, thanks! we can update it in other parts of the app at a later date